db_config/db_connection.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import pkg from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const app = express();
+
+const { Pool } = pkg;
+const pool = new Pool({
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
+  host: process.env.DB_HOST,
+  // port: process.env.PORT,
+  dialect: 'postgres'
+});
+
```

Comment:
Great let me propose something. on initial render of your project, it would make sense if we made it such that the tables required by our application are created automatically. 
`pool.connect((err) => {
if(err){...} else {
pool.query('create table tablename if not exists(...))} etc.`
that way once you application is deployed, it does not break on start because the database will not contain any relations.

Reply:
sir i do not really understand this sir.